src/components/Tooltip/TooltipPlacement.ts

```diff
@@ -7,3 +9,17 @@ export type TooltipPlacement =
     | 'top-right'
     | 'center-left'
     | 'center-right';
+
+const tooltipPlacementMap = {
+    'bottom-left': 'bottom-start',
+    'bottom-center': 'bottom',
+    'bottom-right': 'bottom-end',
+    'top-left': 'top-start',
+    'top-center': 'top',
+    'top-right': 'top-end',
+    'center-left': 'left',
+    'center-right': 'right'
+};
+
+export const mapTooltipPlacementToPopperPlacement = (tooltipPlacement: TooltipPlacement | Placement): Placement =>
```

Comment:
Do you see an advantage to having this function vs. accessing it from the map object directly?

Reply:
@dornelasN Let's check this